api/composer.json

```diff
@@ -8,7 +8,6 @@
         "api-platform/core": "2.6.6",
         "composer/package-versions-deprecated": "1.11.99",
         "cweagans/composer-patches": "1.7.1",
-        "doctrine/annotations": "1.13.2",
```

Comment:
In welchem package sind die attribute?

Reply:
Direkt in doctrine/orm enthalten (ab 2.9)